app/controllers/post_controller.rb

```diff
@@ -0,0 +1,19 @@
+class PostController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @user = User.find(params[:user_id])
+    @post = @user.posts.new(get_params)
+    @posts = @list.tasks.all
```

Comment:
devise has a helper called current_helper, so can just do current_user.posts.new instead of defining @user okie!
also note the extra line, @posts= @list.task.all 😅